crates/test-framework/src/spiced/mod.rs

```diff
@@ -202,7 +202,7 @@ impl SpicedInstance {
             };
             nix::sys::signal::kill(
                 nix::unistd::Pid::from_raw(pid_i32),
-                nix::sys::signal::Signal::SIGTERM,
+                nix::sys::signal::Signal::SIGKILL,
```

Comment:
This should stay SIGTERM to allow the instance to gracefully shutdown. It will be killed when the instance is dropped.